src/conn/conn_compact.c

```diff
@@ -498,8 +591,12 @@ __wt_compact_server_create(WT_SESSION_IMPL *session)
     FLD_SET(conn->server_flags, WT_CONN_SERVER_COMPACT);
 
     WT_RET(__wt_calloc_def(session, conn->hash_size, &conn->background_compact.compacthash));
-    for (i = 0; i < conn->hash_size; i++)
+    WT_RET(
+      __wt_calloc_def(session, conn->hash_size, &conn->background_compact.compactexcludeentryhash));
+    for (i = 0; i < conn->hash_size; i++) {
         TAILQ_INIT(&conn->background_compact.compacthash[i]);
```

Comment:
compacthash could be renamed something else. We need not refer "compact" again, as it is part of the background compact structure

Reply:
Yes, @Sean04 voted for `stathash`, will use that instead.

Reply:
Updated in 434d9240edd6785d13ac2bad8187db7b03de9bef